README.md

```diff
@@ -71,9 +71,28 @@ Without the `\` in front of `EOF`, `$HOME` and `$PATH` would be interpreted befo
 
 * in your final strigo script, make sure to **install docker** (which adds the ubuntu user to the docker group) **before any other service bound to interact with docker**. For example: install `code-server` after `docker` if you plan to use an extension to manage `docker` repositories, images and containers; in this case the ubuntu user is already in the `docker` group when `code-server` is launched (and the extension works properly)
 
-* the cloud-init output log files capture console outputs while the strigo labs VM initializes, so it can help debugging your initialization script following a launch if the instance does not behave the way you intended:
+* the log files of the cloud-init capture outputs while the strigo labs VM initializes, so it can help debugging your initialization script following a launch if the instance does not behave the way you intended:
 
 ```sh
 tail -n 20 /var/log/cloud-init-output.log
 tail -n 20 /var/log/cloud-init.log
 ```
+
```

Comment:
Maybe we could create separate doc files for each script doc and reference them from this README?

Reply:
good idea, I moved the documentation in a documentation file alongside the script